apps/jenkins/src/main/fabric8/openshift-deployment.yml

```diff
@@ -71,7 +75,7 @@ spec:
               key: token
         resources:
             limits:
-              memory: 1Gi
+              memory: 512Mi
```

Comment:
I'm not sure this will work too well because OSO uses this to calculate the request resource values which if too low means jenkins didn't start, although the other settings may help with that I guess - fingers crossed

Reply:
FWIW, jenkins starts fine with this. I still need to look some more and do actual measurements.

Reply:
512Mi is still technically the default OOTB with the openshift/origin template for jenkins 

our online environment motivates this as much as anything (the default memory allotment for namespaces for the free tier is only 2Gi 

but I can say that in our extended tests that leverage jenkins pipelines via openshift pipeline build strategy, we had to bump this up;  and the reason was *NOT* java heap; it was all the `sh ...` invocations from the pipeline scripts (`oc` most notably) that was impacting overall container memory usage and leading to flakiness